docs/UserGuide.md

```diff
@@ -176,56 +245,45 @@ Format: `add-picture INDEX FILE_PATH`
 Examples:
 * `add-picture 1 /Users/john/Desktop/jake.png`
 
-### Listing all persons : `list`
-
-Shows a list of all persons in FriendDex.
+### Clearing all entries : `clear`
 
-<div markdown="span" class="alert alert-primary">:bulb: **Tip:**
-You can optionally provide a group name to list all friends in that group.
-</div>
+Clears all entries from FriendDex.
 
-Format: `list [n/GROUP_NAME]`
+Format: `clear`
 
-### Editing a person : `edit`
+### Deleting a person : `delete`
 
-Edits an existing person in FriendDex.
+Deletes the specified person from FriendDex.
 
-Format: `edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [b/DATETIME] [t/TAG]…​`
+Format: `delete INDEX`
 
-* Edits the person at the specified `INDEX`.
-* At least one of the optional fields must be provided.
-* Existing values will be updated to the input values.
-* When editing tags, the existing tags of the person will be removed i.e. adding of tags is not cumulative.
-* You can remove all the person’s tags by typing `t/` without
-    specifying any tags after it.
+* Deletes the person at the specified `INDEX`.
 
 Examples:
-*  `edit 1 p/91234567 e/johndoe@example.com` Edits the phone number and email address of the 1st person to be `91234567` and `johndoe@example.com` respectively.
-*  `edit 2 n/Betsy Crower t/` Edits the name of the 2nd person to be `Betsy Crower` and clears all existing tags.
+* `list` followed by `delete 2` deletes the 2nd person in FriendDex.
+* `find Betsy` followed by `delete 1` deletes the 1st person in the results of the `find` command.
 
-### Viewing full details : `details`
+### Deleting a special date : `del-date`
 
-Displays the full details of the specified person from FriendDex.
+Deletes a special date from an existing person in FriendDex.
 
-Format: `details INDEX`
+Format: `del-date INDEX i/DATE_INDEX`
 
-* Displays the full details of the person at the specified `INDEX`.
+* Deletes a special date with the person at the specified `INDEX`.
 
 Examples:
-* `list` followed by `details 2` displays the details of the 2nd person in FriendDex.
-* `find Betsy` followed by `details 1` displays the details of the 1st person in the results of the `find` command.
+* `del-date 1 i/2` deletes the 2nd special date from the 1st listed friend.
 
-### Recording a meeting : `add-meeting`
+### Deleting a friend group: `del-group`
```

Comment:
Just make sure to change the command word in the program beforehand.

Reply:
Yeap done